src/studio/src/designer/backend/Designer.csproj

```diff
@@ -78,6 +78,8 @@
     <CoreLanguage Include="../Common/Languages/ini/*" />
     <Compile Remove="Controllers\ManualTestingController.cs" />
     <None Remove="jwtselfsignedcert.pfx" />
+    <Watch Remove="Controllers\RepositorySettingsController.cs" />
+    <Watch Remove="Enums\DatamodellingPreference.cs" />
```

Comment:
Could you explain the purpose of this?

Reply:
Hm, that should probably not have been included - at least I haven't added it on purpose. It removes those specific .cs files from being watched for file changes. The Watch element in the ItemGroup above adds all .cs files, so no reason to exclude these. I'll remove those two lines. Thx.